File: apps/file-storage/src/main/java/dev/felipeflohr/w2gservices/filestorage/controllers/HealthcheckController.java
package dev.felipeflohr.w2gservices.filestorage.controllers;

import jakarta.ws.rs.GET;
import jakarta.ws.rs.Path;
import jakarta.ws.rs.core.Response;

@Path("/healthcheck")
public class HealthcheckController {
    @GET
    public Response healthcheck() {
        return Response.noContent().build();
    }
}

File: apps/builder/src/main/kotlin/dev/felipeflohr/w2gservices/builder/controllers/healthcheck/HealthcheckController.kt
package dev.felipeflohr.w2gservices.builder.controllers.healthcheck

import org.springframework.http.HttpStatus
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.ResponseStatus
import org.springframework.web.bind.annotation.RestController

@Suppress("EmptyMethod")
@RestController
@RequestMapping("/healthcheck")
class HealthcheckController {
    @GetMapping
    @ResponseStatus(value = HttpStatus.OK)
    fun health() {}
}
